internal/pkg/cli/svc_logs.go

```diff
@@ -61,6 +61,9 @@ type wkldLogOpts struct {
 	sel         deploySelector
 	logsSvc     logEventsWriter
 	initLogsSvc func() error // Overriden in tests.
+
+	// cached variables.
+	targetEnv *config.Environment
```

Comment:
huh why does staticcheck say it's unused 🤔 it's assigned in `getTargetEnv()` below

Reply:
It happened to 3288# as well 🤔